src/Platforms/OraclePlatform.php

```diff
@@ -462,9 +462,10 @@ public function getListTableIndexesSQL($table, $database = null)
                            SELECT ucon.constraint_type
                            FROM   user_constraints ucon
                            WHERE  ucon.index_name = uind_col.index_name
+                             AND  ucon.table_name = " . $table . '
```

Comment:
I'd try not to duplicate the parameters. Could this be replaced with the following?
```sql
AND  ucon.table_name = uind_col.table_name
```

Reply:
Ah ok, yeah that makes sense, I will test that.